javaee/faces/src/main/resources/scaffold/faces/pageTemplate.xhtml

```diff
@@ -22,7 +22,7 @@
 				</h:link>
 				<div class="nav-collapse collapse">
 					<ul class="nav">
-						<li><a href="http://forge.jboss.org/docs/important_plugins/ui-scaffolding.html">How to Customize</a></li>
+						<li><a href="http://forge.jboss.org/document/hands-on-lab">How to Customize</a></li>
```

Comment:
yep. couldn't find the right plugin documentation, so I replaced the broken link with one to the hands-on lab which is IMHO the closest to what I would expect behind a "How to Customize" link.
If you know of some better link, just reject this PR :-)